modules/t/strainSlice.t

```diff
@@ -43,15 +43,6 @@ my $strain_slice = Bio::EnsEMBL::Variation::StrainSlice->new(
 
 my $vf_adaptor = $vdba->get_VariationFeatureAdaptor;
```

Comment:
This can be removed as it is not used anymore.

Reply:
It is used further down line 83.

Reply:
indeed, my bad.